sys/arm64/arm64/trap.c

```diff
@@ -347,30 +347,27 @@ data_abort(struct thread *td, struct trapframe *frame, uint64_t esr,
 		panic("data abort in critical section or under mutex");
 	}
 
+	switch (ESR_ELx_EXCEPTION(esr)) {
+	case EXCP_INSN_ABORT:
+	case EXCP_INSN_ABORT_L:
+		ftype = VM_PROT_EXECUTE;
+		break;
+	default:
+		if (esr & ISS_DATA_WnR) {
+			ftype = VM_PROT_WRITE;
 #if __has_feature(capabilities)
-	if ((esr & ISS_DATA_DFSC_MASK) == ISS_DATA_DFSC_LC_SC) {
-		sig = SIGSEGV;
-		ucode = (esr & ISS_DATA_WnR) == 0 ? SEGV_LOADTAG :
-		    SEGV_STORETAG;
-		error = KERN_FAILURE;
-	} else
+			if ((esr & ISS_DATA_DFSC_MASK) ==
+				ISS_DATA_DFSC_LC_SC)
```

Comment:
This should fit just fine on one line (78 < 80), and if you were to wrap it, it should only have 4 spaces for the continuation line